optionals/tracers/data/tracer_white.rvmat

```diff
@@ -0,0 +1,18 @@
+////////////////////////////////////////////////////////////////////
```

Comment:
comment needed?

Reply:
I copied it all from the yellow rvmat, but they all have the mikero tools comment left in.
Don't know why they were left in, but i'm happy to remove them if you think that's best.

Reply:
yes they can get removed